cmd/gtfToBed/gtfToBed.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/common"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+	"strings"
+)
+
+func gtfToBed(fileName string, outFile string) {
+	file := fileio.EasyOpen(fileName)
+	defer file.Close()
```

Comment:
Might be good to move the closes to the end and check the error that they return.